zimbra/src/main/java/fr/openent/zimbra/service/data/sqlTaskServices/SqlICalTaskService.java

```diff
@@ -0,0 +1,111 @@
+package fr.openent.zimbra.service.data.sqlTaskServices;
+
+import fr.openent.zimbra.core.constants.Field;
+import fr.openent.zimbra.core.enums.TaskStatus;
+import fr.openent.zimbra.helper.ConfigManager;
+import fr.openent.zimbra.helper.PromiseHelper;
+import fr.openent.zimbra.model.action.Action;
+import fr.openent.zimbra.model.task.ICalTask;
+import fr.openent.zimbra.service.DbTaskService;
+import fr.wseduc.webutils.http.Renders;
+import io.vertx.core.Future;
+import io.vertx.core.Promise;
+import io.vertx.core.Vertx;
+import io.vertx.core.json.JsonArray;
+import io.vertx.core.json.JsonObject;
+import io.vertx.core.logging.Logger;
+import io.vertx.core.logging.LoggerFactory;
+import org.entcore.common.sql.Sql;
+import org.entcore.common.sql.SqlResult;
+
+public class SqlICalTaskService extends DbTaskService<ICalTask> {
+    protected static final Logger log = LoggerFactory.getLogger(Renders.class);
+    private final String icalTable = "ical_request_tasks";
```

Comment:
peut être static puis ne suis pas la convention nommage static final en java :  

```suggestion
      private static final String I_CAL_TABLE = "ical_request_tasks";
```